pkg/event/target/scram_client.go

```diff
@@ -0,0 +1,36 @@
+package target
+
+import (
+	"crypto/sha256"
```

Comment:
you can use our optimized sha256 `github.com/minio/sha256-simd`

Reply:
Updated